code/modules/mob/abstract/observer/observer.dm

```diff
@@ -311,6 +311,16 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		medHUD = 1
 		to_chat(src, "<span class='notice'><B>Medical HUD Enabled</B></span>")
 
+/mob/abstract/observer/verb/scan_target()
+	set category = "Ghost"
+	set name = "Medical Scan Target"
+	set desc = "Analyse the health of whatever you are following."
+
+	if(ishuman(following))
+		health_scan_mob(following, usr, TRUE, TRUE)
+
+	else to_chat(src, "<span class='notice'>Not a scannable target.</span>")
```

Comment:
please make this indented under the else. also, make it give different messages. for instance, if i'm following a non-human, it gives me the current one, if i'm not following anyone, it says something like "Not following a target!"

Reply:
also replace the "<span class='notice'>Not a scannable target.</span>" stuff with SPAN_WARNING("Not a scannable target.")

Reply:
I implemented these, I think!